### ISSUE TITLE
Migration 0004 fails on existing db

### ISSUE THREAD

**Issue by gabn88** [dokterbob/django-newsletter#253]
It seems it does not like the 

```
def get_default_newsletter():
    return Newsletter.get_default()
```

Traceback:
```
 Applying newsletter.0004_auto_20180407_1043...Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 204, in handle
    fake_initial=fake_initial,
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/migrations/executor.py", line 115, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/migrations/executor.py", line 145, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/migrations/migration.py", line 129, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/migrations/operations/fields.py", line 221, in database_forwards
    schema_editor.alter_field(from_model, from_field, to_field)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/backends/base/schema.py", line 531, in alter_field
    old_db_params, new_db_params, strict)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/backends/base/schema.py", line 635, in _alter_field
    new_default = self.effective_default(new_field)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/backends/base/schema.py", line 245, in effective_default
    default = field.get_db_prep_save(default, self.connection)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/models/fields/related.py", line 963, in get_db_prep_save
    return self.target_field.get_db_prep_save(value, connection=connection)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 770, in get_db_prep_save
    prepared=False)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 958, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/home/gerben/ProductionEquinem/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 966, in get_prep_value
    return int(value)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'Newsletter'
```

I will get back to this issue when I have more info.

**Comment by gabn88** [comment]
The problem is that get_Default_newsletter gives back a newsletter instance and not a pk. I have fixed it by giving back a pk, or None, so that the migration will always succeed.

**Comment by gabn88** [comment]
It would be good to have my fix merged, because this is the last remaining hurdle for upgrading to Django 2.0 :) Thanks a lot!

**Comment by dokterbob** [comment]
Fixed in #254
